pkg/env/grpc_max_message_size.go

```diff
@@ -0,0 +1,6 @@
+package env
+
+var (
+	// MaxMsgSizeSetting is the setting used for gRPC servers and clients to set maximum receive sizes.
+	MaxMsgSizeSetting = RegisterIntegerSetting("ROX_GRPC_MAX_MESSAGE_SIZE", 24*1024*1024)
```

Comment:
Go noob question here, but should we define a constant for this somewhere, rather than having a hard-coded value here, and constants still over in `pkg/grpc/server.go`? 

I can see splitting the sources of truth confusing someone when they revisit the code later.